tmt-app/src/Components/BobaBot.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import BobaBotLarge from "../Images/art/boba_bot_Large.svg";
+import { Box, Typography } from "@mui/material";
+
+function BobaBot({ text }) {
+  return (
+    <Box
+      sx={{
+        alignItems: "center",
+        display: "inline-flex",
+        position: "relative",
+      }}
+    >
+      <img src={BobaBotLarge} alt={"Mascot Icon"} />
+      <Box
+        sx={{
+          alignItems: "center",
+          display: "flex",
+          height: "40%",
+          left: "15%",
+          position: "absolute",
+          top: "10%",
+          width: "70%",
+        }}
+      >
+        <Typography
+          sx={{
+            fontWeight: "bold",
+            textAlign: "center",
+          }}
+        >
```

Comment:
Does this need a variant?

Reply:
I didn't see any specifications for variants in the Specs for Devs file in Figma? Should defining the variants for text be something the devs do?